### PR TITLE
[FIX][product_supplierinfo_discount] Fail when same products has mult…

### PR DESCRIPTION
**Pull request by andrius-preimantas**
…iple records of same supplier

Odoo does not restrict entering mulltiple product.supplierinfo records
for the same product and same partner.

### CODE REVIEW

File: product_supplierinfo_discount/models/purchase_order.py [hunk 1]
```diff
@@ -51,7 +51,10 @@ def onchange_product_id(
              ('name', 'child_of', partner.commercial_partner_id.id)])
         if not sinfos:
             return res
-        seller_uom = sinfos.product_uom.id or False
+        if len(sinfos) > 1:
```
Comment (pedrobaeza) [inline product_supplierinfo_discount/models/purchase_order.py]: Please change this to `seller_uom = sinfos[:1].product_uom.id`. It makes the same and it doesn't fail on empty values (not on v8).

Comment (andrius-preimantas) [inline product_supplierinfo_discount/models/purchase_order.py]: Cool, thanks!

**Comment by max3903** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**